examples/gptreview-ghaction/README.md

```diff
@@ -0,0 +1,21 @@
+# GPTReview
+
+This folder contains an example of building and implementing your own code reviewer as part of GitHub Actions. 
+
+Below are the files present here:
+
+- `codereview.gpt`: Contains the GPTScript code and prompts.
+- `workflow.yaml`: The workflow file for the GitHub action.
+
+## Pre-requisites
+
+- GitHub Account
+- OpenAI API Key
+
+## How To Run This Example
+
+- Create a new repository in your GitHub account and create a `codereview.gpt` file in that repo based on the contents provided in this file.
```

Comment:
Call out that is in the root of the repo.

Reply:
Done.